frontend/packages/console-app/src/components/modals/clone/clone-pvc-modal.tsx

```diff
@@ -103,7 +112,8 @@ const ClonePVCModal = withHandlePromise((props: ClonePVCModalProps) => {
         namespace: resource.metadata.namespace,
       },
       spec: {
-        storageClassName: resource.spec.storageClassName,
+        // eslint-disable-next-line object-shorthand
+        storageClassName: storageClassName || resource.spec?.storageClassName,
```

Comment:
why `|| resource.spec?.storageClassName` part is needed ??

Reply:
In case if user does not select the storage class, it uses the same store class name of the one selected for clone

Reply:
why are u having this assumption ?? Isn't StorageClass a "required" field ??